values.yaml

```diff
@@ -26,6 +26,7 @@ serviceAccount:
 pod:
 # Configure pod annotations
   annotations: {}
+  extraLabels: {}
```

Comment:
@JRHerlehy Thank you for your contribution. Please can you update the variable name to `labels` instead of `extraLabels`?